config/routes.rb

```diff
@@ -35,6 +35,9 @@
   post 'i-forgor/commit/:password_reset_token', to: 'users#commit_apply_new_password', as: 'forgor_apply_commit'
   get 'logout', to: 'session#destroy', as: 'logout'
   get 'browse', to: 'links#browse'
+  get 'links/:id/history', to: 'links#history'
```

Comment:
ignore this garbage, once i move history into its own controller this is going to get replaced with a proper resource with its own path helpers